scripts/ci-build-azure-ccm.sh

```diff
@@ -80,7 +80,8 @@ can_reuse_artifacts() {
     done
 
     if [[ -n "${TEST_WINDOWS:-}" ]]; then
-        if ! docker manifest inspect "${REGISTRY}/${CNM_IMAGE_NAME}:${IMAGE_TAG}" | grep -q "\"os.version\": \"${WINDOWS_IMAGE_VERSION}\""; then
+        FULL_VERSION=$(docker manifest inspect mcr.microsoft.com/windows/nanoserver:${WINDOWS_IMAGE_VERSION} | jq -r '.manifests[0].platform["os.version"]')
```

Comment:
This change will require a call out to mcr.microsoft.com, are we O.K. with that?

Can you explain how `$WINDOWS_IMAGE_VERSION` will be different from the `os.version` value in the CNM image? Why can't we use it as-is?

Reply:
Thanks for reply. 
Because in docker manifest, os.version like "1809" ($WINDOWS_IMAGE_VERSION) cannot be recognized. Once the os.version is set to "1809", the image cannot be pulled successfully. FULL_VERSION is like an official windows os version. The corresponding FULL_VERSION of "1809" is "10.0.17763.3165".
If the call to mcr.microsoft.com is not good, I think we can hard code the FULL_VERSION.
(This PR can also help answer the question https://github.com/kubernetes-sigs/cloud-provider-azure/pull/2026)

Reply:
`$WINDOWS_IMAGE_VERSION` will be set to either `windows-2019` or `windows-2022`.
The image tags for the nanosever image are `1809` for Windows Server 2019 and `ltsc-2022` for Windows Server 2022.
I don't think these changes will work.

Reply:
Whoops, nevermind, i confused `$WINDOWS_IMAGE_VERSION` win `$WINDOWS_SERVER_VERSION` 
I think these changes will be OK.